{"uid":"6dd01aabd621b15b","name":"Check App Dialod popup","fullName":"Check App Dialod popup","historyId":"e783d62f93e831a5381735c103042c6f","time":{"start":1675358119498,"stop":1675358120282,"duration":784},"status":"failed","statusMessage":"The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > Loading script failed.\n(error: http://localhost:3001/remoteEntry.js)\nwhile loading \"./Button\" from webpack/container/reference/component-app\nwhile loading \"./Dialog\" from webpack/container/reference/component-app\nwhile loading \"./ToolTip\" from webpack/container/reference/component-app\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","statusTrace":"ScriptExternalLoadError: The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > Loading script failed.\n(error: http://localhost:3001/remoteEntry.js)\nwhile loading \"./Button\" from webpack/container/reference/component-app\nwhile loading \"./Dialog\" from webpack/container/reference/component-app\nwhile loading \"./ToolTip\" from webpack/container/reference/component-app\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application\n    at webpack/container/reference/component-app (http://localhost:3002/main.js:11:25)\n    at __webpack_require__ (http://localhost:3002/main.js:72:41)\n    at handleFunction (http://localhost:3002/main.js:248:31)\n    at <unknown> (http://localhost:3002/main.js:267:15)\n    at Array.forEach (<anonymous>)\n    at __webpack_require__.f.remotes (http://localhost:3002/main.js:230:36)\n    at <unknown> (http://localhost:3002/main.js:113:40)\n    at Array.reduce (<anonymous>)\n    at __webpack_require__.e (http://localhost:3002/main.js:112:67)\n    at <unknown> (http://localhost:3002/main.js:424:21)","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[],"testStage":{"status":"failed","statusMessage":"The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > Loading script failed.\n(error: http://localhost:3001/remoteEntry.js)\nwhile loading \"./Button\" from webpack/container/reference/component-app\nwhile loading \"./Dialog\" from webpack/container/reference/component-app\nwhile loading \"./ToolTip\" from webpack/container/reference/component-app\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","statusTrace":"ScriptExternalLoadError: The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > Loading script failed.\n(error: http://localhost:3001/remoteEntry.js)\nwhile loading \"./Button\" from webpack/container/reference/component-app\nwhile loading \"./Dialog\" from webpack/container/reference/component-app\nwhile loading \"./ToolTip\" from webpack/container/reference/component-app\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application\n    at webpack/container/reference/component-app (http://localhost:3002/main.js:11:25)\n    at __webpack_require__ (http://localhost:3002/main.js:72:41)\n    at handleFunction (http://localhost:3002/main.js:248:31)\n    at <unknown> (http://localhost:3002/main.js:267:15)\n    at Array.forEach (<anonymous>)\n    at __webpack_require__.f.remotes (http://localhost:3002/main.js:230:36)\n    at <unknown> (http://localhost:3002/main.js:113:40)\n    at Array.reduce (<anonymous>)\n    at __webpack_require__.e (http://localhost:3002/main.js:112:67)\n    at <unknown> (http://localhost:3002/main.js:424:21)","steps":[{"name":"\"before each\" hook","time":{"start":1675358119499,"stop":1675358119833,"duration":334},"status":"passed","steps":[{"name":"visit http://localhost:3002","time":{"start":1675358119600,"stop":1675358119833,"duration":233},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false}],"attachments":[],"parameters":[],"stepsCount":1,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":true},{"name":"get button","time":{"start":1675358119835,"stop":1675358120281,"duration":446},"status":"failed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"uncaught exception ScriptExternalLoadError: Loading script failed.\n(error: http://localhost:3001/remoteEntry.js)\nwhile loading \"./Button\" from webpack/container/reference/component-app\nwhile loading \"./Dialog\" from webpack/container/reference/component-app\nwhile loading \"./ToolTip\" from webpack/container/reference/component-app","time":{"start":1675358120281,"stop":1675358120281,"duration":0},"status":"failed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false}],"attachments":[{"uid":"c97be7b0fb826243","name":"e2e/checkApplication.cy.ts:2023-02-02T17:15:19.185Z","source":"c97be7b0fb826243.png","type":"image/png","size":166503},{"uid":"d2c0010e3ded26bb","name":"e2e/checkApplication.cy.ts:2023-02-02T17:15:19.937Z:attempt-1","source":"d2c0010e3ded26bb.png","type":"image/png","size":167856},{"uid":"24f73d289ffe332b","name":"video recording","source":"24f73d289ffe332b.mp4","type":"video/mp4","size":1788761}],"parameters":[],"stepsCount":4,"attachmentsCount":3,"shouldDisplayMessage":true,"hasContent":true},"afterStages":[],"labels":[{"name":"suite","value":"Check App1"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Product defects","matchedStatuses":[],"flaky":false}],"tags":[]},"source":"6dd01aabd621b15b.json","parameterValues":[]}